django-web-app/cogniguard/urls.py

```diff
@@ -19,22 +17,10 @@
     path('admin/', admin.site.urls),
     # path("logout/", RedirectView.as_view(url="/admin/logout/")),
     path("", include("home.urls")),
-
-    # for rendering reset password cnf template
-    path('password-reset-confirm/<uidb64>/<token>/',
```

Comment:
Here password-reset-confirm and complete paths are missing, 
see the latest changes 
vercel deployment configuration for static URL patterns are also missing